src/main/webapp/jsp/instructorCourseJoinConfirmation.jsp

```diff
@@ -1,84 +1,29 @@
-    <!--[if lt IE 9]>
-        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-    <![endif]-->
-
-    <script type="text/javascript" src="/js/googleAnalytics.js"></script>
-    <script type="text/javascript" src="/js/jquery-minified.js"></script>
-    <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js"></script>
-    <script type="text/javascript" src="/js/common.js"></script>
-    <script type="text/javascript" src="/bootstrap/js/bootstrap.min.js"></script>
-    
-    <jsp:include page="../enableJS.jsp"></jsp:include>
-</head>
-
-<body>
-    <jsp:include page="<%=Const.ViewURIs.STUDENT_HEADER%>" />
-
-    <div class="container" id="mainContent">
-        <br>
-        <jsp:include page="<%=Const.ViewURIs.STATUS_MESSAGE%>" />
-        <br>
-        <div id="topOfPage"></div>
-
-        <div class="panel panel-primary panel-narrow">
-            <div class="panel-heading">
-                <h3>Confirm your Google account</h3>
-            </div>
-            <div class="panel-body">
-                <p class="lead">
-                    You are currently logged in as <span><strong><%=data.account.googleId%></strong></span>.
-                    <br>If this is not you please <a
-                        href="/logout.jsp">log out</a> and re-login using your own Google account. 
-                        <br>If this is you, please confirm below to complete your registration. <br>
-                <div class="align-center">
-                    <%
-                        String ref = "";
-                        if (data.institute == null) {
-
-                            ref = Const.ActionURIs.INSTRUCTOR_COURSE_JOIN_AUTHENTICATED
-                                    + "?key=" + data.regkey;
-                        } else {
-                            ref = Const.ActionURIs.INSTRUCTOR_COURSE_JOIN_AUTHENTICATED
-                                    + "?key=" + data.regkey + "&"
-                                    + Const.ParamsNames.INSTRUCTOR_INSTITUTION + "="
-                                    + Sanitizer.sanitizeForUri(data.institute);
-                        }
-                    %>
-                    <a href="<%=ref%>" id="button_confirm"
-                        class="btn btn-success">Yes, this is my account</a> <a href="/logout.jsp" id="button_cancel"
-                        class="btn btn-danger">No, this is not my account</a>
-                </div>
-                </p>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib tagdir="/WEB-INF/tags/student" prefix="ts" %>
+
+<c:set var="jsIncludes">
+</c:set>
+
+<ts:studentPage pageTitle="TEAMMATES - Instructor" bodyTitle="" jsIncludes="${jsIncludes}">
```

Comment:
If there are no other js / css files to include, just delete the `c:set` segment and remove it from the parameters passed into the tag.